src/CMakeLists.txt

```diff
@@ -741,6 +741,7 @@ IF(WIN32)
   ENDIF(VISIT_WINDOWS_APPLICATION)
 ENDIF(WIN32)
 
+OPTION(VISIT_CMAKE_VERBOSE_GEN_TARGET_MESSAGES "Display names and details about code gen targets during CMake configure time." OFF)
```

Comment:
This seems to make *messages* during code-gen targets enabled or not. Following up on CMake performance...one way to address it if it is a problem is to also have a variable that conditionally defines code-gen targets only for *dev* builds. That way, users and maintainers would be suffering the possible performance hit

Reply:
It takes quite a long time to configure and generate visit using cmake already. 

I didn't measure how much more time this adds.  I guess we could add an option to completely turn off creating these targets and measure.

My observation from this exercise:
We don't have good hygiene with respect to applying all of our code gen tools consistently. 
These CMake targets can help a ton with this, I think its worth it to have them on.

Reply:
I didn't mean to sugges we don't do this...only that we should maybe consider additional controls w.r.t. to when these targets get created.